api/models/collectionModel.js

```diff
@@ -20,9 +20,28 @@ const Collection = db.define(
     },
     linkId: {
       type: Sequelize.INTEGER
+    },
+    filename: {
+      type: Sequelize.STRING
+    },
+    description: {
+      type: Sequelize.STRING
+    },
+    category: {
+      type: Sequelize.STRING
+    },
+    status: {
+      type: Sequelize.BOOLEAN
     }
   },
   { timestamps: true }
 )
 
+Collection.associate = function (models) {
+  Collection.hasMany(models.CollectionUser, {
+    as: 'collection',
```

Comment:
as is alias for collectionuser so it should not say collection as collection is the main table and associated table should hae different alias